utils/api.ts

```diff
@@ -290,3 +290,9 @@ export function fetchBookstoreCMSProductsByTagId(tagId: string, { offset, limit
     },
   })
 }
+
+export function getArweaveAPIEndpoints() {
```

Comment:
```suggestion
export function getArweaveLinkAPIEndpoint() {
```

Reply:
[✏️ Rename function to getArweaveLinkAPIEndpoint](https://github.com/likecoin/3ook-com/pull/47/commits/7cd57532c0ed1d1c7900b3fdd05f08ed17b653e8)